src/bin/rdcore/kargs.rs

```diff
@@ -19,9 +19,21 @@ use libcoreinst::io::*;
 #[cfg(target_arch = "s390x")]
 use libcoreinst::s390x;
 
+#[cfg(target_arch = "s390x")]
+use libcoreinst::blockdev::*;
+
 use crate::cmdline::*;
 use crate::rootmap::get_boot_mount_from_cmdline_args;
 
+#[cfg(target_arch = "s390x")]
+pub fn zipl(config: ZiplConfig) -> Result<()> {
+    if config.boot_mount.is_none() {
+        bail!("--boot-mount required");
```

Comment:
If it's required, then we could just make the field a `String` instead of an `Option<String>`. The reason it's an `Option` in other `rdcore` commands is because we support specifying it as a mount or a block device.